tests/veos.vtb

```diff
@@ -61,11 +61,11 @@ vlab-02
 vlab-03
 
 [sonic]
-vlab-01 ansible_host=10.250.0.101 type=kvm hwsku=Force10-S6000 ansible_password=password ansible_user=admin
-vlab-02 ansible_host=10.250.0.102 type=kvm hwsku=Force10-S6100 ansible_password=password
-vlab-03 ansible_host=10.250.0.105 type=kvm hwsku=Force10-S6000 ansible_password=password ansible_user=admin
-vlab-04 ansible_host=10.250.0.107 type=kvm hwsku=Force10-S6000 ansible_password=password ansible_user=admin
+vlab-01 ansible_host=10.250.0.101 ansible_hostv6=fec0:0:0:0:0:ffff:afa:1 type=kvm hwsku=Force10-S6000 ansible_password=password ansible_user=admin
```

Comment:
simplify the v6 ip address: fec0::ffff:afa:1

Reply:
fixed as per comment.